src/encoded/static/browser.js

```diff
@@ -70,9 +68,9 @@ if (typeof window !== 'undefined' && window.document && !window.TEST_RUNNER) {
         var server_stats = require('querystring').parse(window.stats_cookie);
         var UseApp = connect(store.mapStateToProps)(App);
         var app;
-        
+
         try {
-            app = ReactDOM.render(<Provider store={store}><UseApp /></Provider>, document);
+            app = ReactDOM.hydrate(<Provider store={store}><UseApp /></Provider>, document);
```

Comment:
ReactDOM.render was deprecated in React v16 ( https://reactjs.org/docs/react-dom.html#render )